saleor/graphql/checkout/tests/mutations/test_checkout_billing_address_update.py

```diff
@@ -148,7 +148,7 @@ def test_checkout_billing_address_update_by_id_without_street_address_2(
 )
 def test_checkout_billing_address_update(
     mocked_invalidate_checkout_prices,
-    user_api_client,
+    superuser_api_client,
```

Comment:
`superuser_api_client` is used only in some special cases, since in most cases Saleor tries to relies on permission and not on the `is_superuser` setting (e.g. it's not available to any staff users in Saleor Cloud). I guess we should keep `user_api_client` here, and if we test private metadata we should have the second test with `staff_api_client` and proper permissions granted.

Reply:
there might be a wider discussion if there is a use case for updating private metadata in this call. Also there is a new situation where Address do not exist yet as in `updatePrivateMetadata` mutation